safehttp/incoming_request.go

```diff
@@ -108,3 +109,24 @@ func (r *IncomingRequest) MultipartForm(maxMemory int64) (*MultipartForm, error)
 			}},
 		nil
 }
+
+// Context returns the http.Request Context. This is always non-nil and will
+// default to the background context.
+//
+// For outgoing client requests, the context controls cancellation.
+//
+// For incoming server requests, the context is canceled when the client's
+// connection closes, the request is canceled (with HTTP/2), or when the
+// ServeHTTP method returns.
```

Comment:
Please change this comment so that it fits `safehttp.IncomingRequest` and not `http.Request`. For instance `IncomingRequest` is never used for outgoing client requests.